src/main/java/dota/buff/service/MatchService.java

```diff
@@ -8,12 +8,13 @@
 import java.util.List;
 
 public interface MatchService {
+
     MatchDTO getMatchById(long matchId);
 
-    List<PlayerDTO> getAllMatchPlayers(long matchId);
```

Comment:
What are you think about param matchId compared with current match, what is better ? 
And this concern remain methods like getAllMatchHeroes, getWinner.

Reply:
I think should be better to prepare an entity beforehand -> `getAllMatchHeroes(match);` rather than make it explicitly(work with the state of the object) in the spot of calling needed method and do smth like -> `getAllMatchHeroes(match.getId());`

Reply:
Ok, then we choice the last option